src/helpers/map.js

```diff
@@ -185,9 +126,8 @@ export function traversableCellsInTile(tileIsTraversable, cpt, objRadius) {
  *
  * cpt: number of cells per tile
  * map: 2D array representing the Tagpro map
- * me: reference to the TagproBot
  */
-export function getTraversableCells(cpt, map, me) {
+export function getTraversableCells(cpt, map) {
```

Comment:
I figured it out be a hassle to keep passing in me, so the player's team is just kept track of in player.js and any thing else that needs it can call amBlue()